src/test/java/org/testcontainers/containers/TarantoolCartridgeBootstrapFromLuaWithFixedPortsTest.java

```diff
@@ -21,8 +26,8 @@ public class TarantoolCartridgeBootstrapFromLuaWithFixedPortsTest {
                     "Dockerfile",
                     "cartridge/instances_fixedport.yml",
                     "cartridge/topology_fixedport.lua")
-                    .withCopyFileToContainer(MountableFile.forClasspathResource("cartridge"), "/app")
-                    .withCopyFileToContainer(MountableFile.forClasspathResource("cartridge/instances_fixedport.yml"),"/app/instances.yml")
+                    .withDirectoryBinding("cartridge")
```

Comment:
IIRC Vova changed this line because we had some problems with windows (?) or something. Did you look at the history of this change? Is it safe to use binding here? (I know, it's the default preferred variant, but there was some problem)

Reply:
It's hard to see why it had been changed https://github.com/tarantool/cartridge-java-testcontainers/pull/46. I asked @dkasimovskiy to try but he didn't get what I want and merged PR 🤷‍♂️

Reply:
@akudiyar we will test how it's working with windows. You can check other PRs

Reply:
On the latest WSL2 it works fine.